Packs/Arcanna/Integrations/ArcannaAI/README.md

```diff
@@ -9,11 +9,13 @@ This integration was integrated and tested with version 1.0 and above of Arcanna
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
-    | Server URL | URL of Arcanna API | True |
+    | Server URL (e.g. https://&lt;your arcanna ai api&gt;) | URL of Arcanna API | True |
     | API Key | Api Key for Arcanna API | True |
     | Trust any certificate (not secure) |  | False |
     | Use system proxy settings |  | False |
-    | Default Arcanna Job Id | Default Arcanna Job Id | False |
+    | Default Arcanna Job Id |  | False |
+    | Feature Mapping as an array to map between CLOSING REASON and Arcanna labels |  | False |
+    | Field use to signal to arcanna the status for closing an alert(or marking as feedback for Arcanna) |  | False |
```

Comment:
Please generate this part again after the changes I mentioned in the yml file

Reply:
regenerated.